websites/T/TruckersFM/presence.ts

```diff
@@ -7,14 +7,13 @@ presence.on("UpdateData", () => {
     document.querySelector(".player-artist-text").textContent
   } - ${document.querySelector(".player-title-text").textContent} `;
   const dj = document.querySelector(".live-name").textContent;
-  const liveTill = document.querySelector(".live-time").textContent;
-  const liveTime = liveTill.slice(6);
+  const liveTill = document.querySelector(".live-time") ? document.querySelector(".live-time").textContent.slice(6) : false;
   const presenceData: PresenceData = {
```

Comment:
youd idn't merge consts yet lol

Reply:
Huh?

I merged the liveTime and liveTill in https://github.com/PreMiD/Presences/pull/2867/commits/2fc6d6168ddb64e8ead3dd53f4b33bc8b0fbeb2c

Reply:
Nevermind! I realised what you meant now aha

Reply:
no look

Reply:
![image](https://user-images.githubusercontent.com/9348108/105839563-f9484a00-5fd1-11eb-9253-788a876c77ba.png)

Reply:
these consts are not merged together...